<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="importButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>641</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Import</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>641</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="format">
     <string>%p%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browseButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pathLine">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>519</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>File Path:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="genWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>661</width>
      <height>121</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Please Select Which File To Generate:</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>641</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maxVisibleItems">
      <number>10</number>
     </property>
     <property name="minimumContentsLength">
      <number>100</number>
     </property>
     <item>
      <property name="text">
       <string>---</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PartMaster</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PartRevision</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PartRevisionWithAttachment</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BOO</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BOM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>All</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="confirmButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>641</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirm To Generate</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuNavigate">
    <property name="title">
     <string>Navigate</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Option</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuNavigate"/>
   <addaction name="menuOption"/>
   <addaction name="menuAbout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
